src/utils/circuit-to-svg.tsx

```diff
@@ -0,0 +1,317 @@
+import jscad from "@jscad/modeling"
+import type { AnySoupElement } from "@tscircuit/soup"
+import { su } from "@tscircuit/soup-util"
+import { Footprinter3d } from "jscad-electronics"
+import { convertCSGToThreeGeom, createJSCADRenderer } from "jscad-fiber"
+import { executeJscadOperations, jscadPlanner } from "jscad-planner"
+import { JSDOM } from "jsdom"
+import * as THREE from "three"
+import { BufferGeometry, Float32BufferAttribute } from "three"
+import { OBJLoader } from "three/examples/jsm/loaders/OBJLoader.js"
+import { STLLoader } from "three/examples/jsm/loaders/STLLoader.js"
+import { SVGRenderer } from "three/examples/jsm/renderers/SVGRenderer.js"
+import { createBoardGeomFromSoup } from "../soup-to-3d"
+
+interface CircuitToSvgOptions {
```

Comment:
make sure to put 3d in the filename and this options, and make sure that the "/three" is a separate export